pipelines/tekton/aiedge-e2e/examples/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This ClusterRoleBinding was needed to get the EventListener pod to work. See https://tekton.dev/docs/triggers/eventlisteners/#specifying-the-kubernetes-service-account for more details.